includes/import/WikiImporter.php

```diff
@@ -680,7 +680,7 @@ private function handleLogItem() {
 			$tag = $this->reader->localName;
 
 			if ( !Hooks::run( 'ImportHandleLogItemXMLTag', [
-				$this, $logInfo
+				$this, &$logInfo
```

Comment:
@abador this may potentially break other extensions that use this hook. Comment would be nice to help debug such cases. We are sure this hook is not used in any extensions we use?

Reply:
This could only break the other way around if you want a reference, but you get a copy. STill I'll add a comment